relay/nakama/main.go

```diff
@@ -494,13 +517,14 @@ func logErrorFailedPrecondition(
 	return logError(logger, err, FailedPrecondition)
 }
 
-func errToNakamaError(
-	err error,
-	code int) error {
-	if DebugEnabled {
-		return runtime.NewError(eris.ToString(err, true), code)
+func errToNakamaError(err error, code int) error {
+	if err != nil {
+		if DebugEnabled {
+			return runtime.NewError(eris.ToString(err, true), code)
+		}
+		return runtime.NewError(err.Error(), code)
 	}
-	return runtime.NewError(eris.Errorf("error: %v", err).Error(), code)
+	return nil
 }
```

Comment:
i belive this was the cause of some odd errors the client was seeing where it would return

"error: 0035nil0035"